src/Controls/src/Core/Shell/Shell.cs

```diff
@@ -94,77 +121,215 @@ static void OnFlyoutItemIsVisibleChanged(BindableObject bindable, object oldValu
+		/// </summary>
+		/// <param name="obj">The object that modifies the width of the flyout.</param>
+		/// <param name="value">Defines the width of the flyout.</param>
 		public static void SetFlyoutWidth(BindableObject obj, double value) => obj.SetValue(FlyoutWidthProperty, value);
 
+		/// <summary>
+		/// Gets the height of the flyout.
+		/// </summary>
+		/// <param name="obj">The object that modifies the height of the flyout.</param>
+		/// <returns>The height of the flyout.</returns>
 		public static double GetFlyoutHeight(BindableObject obj) => (double)obj.GetValue(FlyoutHeightProperty);
-		public static void SetFlyoutHeight(BindableObject obj, double value) => obj.SetValue(FlyoutHeightProperty, value);
 
+		/// <summary>
+		/// Sets the height of the flyout.
+		/// </summary>
+		/// <remarks>
+		/// The height of the flyout can be customized by setting the Shell.FlyoutHeight attached properties to double value.
+		/// This enables scenarios such as reducing the height of the flyout so that it doesn't obscure the tab bar.
+		/// </remarks>
+		/// <param name="obj">The object that modifies the height of the flyout.</param>
+		/// <param name="value">Defines the height of the flyout.</param>
+		public static void SetFlyoutHeight(BindableObject obj, double value) => obj.SetValue(FlyoutHeightProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetNavBarIsVisible']/Docs/*" />
+		/// <summary>
+		/// Gets if the navigation bar is visible when a page is presented. 
+		/// </summary>
+		/// <param name="obj">The object that that gets the navigation bar visibility.</param>
+		/// <returns><see langword="true"/> if the navigation bar is visible; otherwise, <see langword="false"/>.</returns>
 		public static bool GetNavBarIsVisible(BindableObject obj) => (bool)obj.GetValue(NavBarIsVisibleProperty);
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='SetNavBarIsVisible']/Docs/*" />
+
+		/// <summary>
+		/// Controls if the navigation bar is visible when a page is presented. 
+		/// By default the value of the property is <see langword="true"/>.
+		/// </summary>
+		/// <param name="obj">The object that modifies the navigation bar visibility.</param>
+		/// <param name="value"><see langword="true"/> to set the navigation bar as visible; otherwise, <see langword="false"/>.</param>
 		public static void SetNavBarIsVisible(BindableObject obj, bool value) => obj.SetValue(NavBarIsVisibleProperty, value);
 
-		/// <include file="../../../docs/Microsoft.Maui.Controls/Shell.xml" path="//Member[@MemberName='GetNavBarHasShadow']/Docs/*" />
+		/// <summary>
+		/// Gets a value that represents if the navigation bar has a shadow.
+		/// </summary>
+		/// <param name="obj">The object that modifies if the navigation bar has a shadow.</param>
+		/// <returns>Returns if the navigation bar has a shadow.</returns>
```

Comment:
I suggest using the standard formulation "<see langword="true"/> if the navigation bar has a shadow when <paramref ="obj"/> is presented, <see langword="false"/> otherwise.